src/projects.rs

```diff
@@ -67,6 +98,33 @@ fn fetch_next_item(config: &Config, project_name: &str) -> Result<Option<Item>,
     Ok(items.first().map(|item| item.to_owned()))
 }
 
+/// Fetch projects and prompt to add them to config one by one
+pub fn import(config: Config) -> Result<String, String> {
+    let options = vec!["add", "skip"];
+    let projects = request::projects(&config)?;
+    let project_ids: Vec<String> = config.projects.values().map(|v| v.to_string()).collect();
+    let new_projects: Vec<Project> = projects
+        .into_iter()
+        .filter(|p| !project_ids.contains(&p.id))
+        .collect();
+    for project in new_projects {
+        println!("{}", project.fmt());
+        match config::select_input("Select an option", options.clone()) {
+            Ok(string) => {
+                if string == "add" {
+                    let modified_config = Config::load(&config.path)?;
```

Comment:
why do we need load config if we provide one as a parameter?
maybe just provide the mutable one?

Reply:
Haha, this is a great callout. I'm used to working with Elixir where there is no mutability so up until now I have just been using return values, but this is an excellent example of where that strategy falls down.

Fixed to make config mutable, compiler is a huge help.